railgun/hack/generators/controllers/networking/main.go

```diff
@@ -234,14 +267,66 @@ var controllerTemplate = `
 type {{.PackageAlias}}{{.Type}}Reconciler struct {
 	client.Client
 
+{{- if or .AcceptsIngressClassNameSpec .AcceptsIngressClassNameAnnotation}}
```

Comment:
If you put a blank line between `IngressClassName` and other fields, you won't need to be concerned with nice block formatting of these other fields, and the generated code will still pass `gofmt`.

Reply:
:+1:  1d99e3614a8fb3eb836028ccb88768ce1ae239e2